README.md

```diff
@@ -1,22 +1,158 @@
+
+```
+curl -fsSL https://raw.githubusercontent.com/Virtomize/virtomize/master/install.sh | bash
+```
+
+## Installation using virtual appliance
+
+### Installation
+
+Download the [latest virtual appliance](https://virtomize.com/downloads)
+
+Follow the import procedure described in our [documentation](https://docu.virtomize.com/#import-ova).
+
+## Manual installation using git 
+
+Recommended only for advanced users for running Virtomize on servers with multiple other services.
+
+***We do not provide OS level support for custom installations***
+
+### Requirements
+
+The following software is required to install and run [Virtomize](https://virtomize.com) on your server. 
+
+- docker
+- docker-compose
+- apache or nginx
+- git
+
+For more information please consult our [documentation page](https://docu.virtomize.com).
+Make sure the server running our software can request your hypervisor API.
+
+we use a simple docker-compose setup to make it as easy as possible.
+This will start our service on `127.0.0.1:8000`
+
+```
+cd /opt
+git clone github.com/virtomize/virtomize
+cd virtomize
+docker-compose up -d
+```
+
+### Cron for update via ui
+
+To enable the update button in the UI add the following cron.
+
+```
+cat <<EOF >/etc/cron.d/vtmz-update
+* * * * * root /opt/virtomize/update.sh
+EOF
+```
+
+### Reverse proxy configuration
+
+To make sure our service can be accessed from your servers domain or IP address including SSL create a reverse proxy using Apache or Nginx.
+
+#### Apache
+
+debian 10 example
+```
+apt -y install apache2
+
+systemctl enable apache2
+a2enmod ssl
+a2enmod proxy
+a2enmod proxy_http
+a2enmod headers
+a2enmod rewrite
+# default config is configured to use the snakeoil cert generated by apache on install 
+# if you want to change this change the apache config
+cp /opt/virtomize/proxy/apache.conf /etc/apache2/sites-available/000-default.conf
+# add dhparm
+curl https://ssl-config.mozilla.org/ffdhe2048.txt >> /etc/ssl/certs/ssl-cert-snakeoil.pem
+systemctl restart apache2
+```
+[apache config](proxy/apache.conf)
+
+#### Nginx
+
+debian 10 example
+```
+apt -y install nginx
+systemctl enable nginx
+# create a certificate or use your own
+# if you want to use a different path change the nginx config
+mkdir /etc/nginx/ssl
+openssl req -x509 -nodes -days 3650 -newkey rsa:2048 -keyout /etc/nginx/ssl/nginx.key -out /etc/nginx/ssl/nginx.crt
+# add dhparm
+curl https://ssl-config.mozilla.org/ffdhe2048.txt > /etc/nginx/ssl/dhparam
+cp /opt/virtomize/proxy/nginx.conf /etc/nginx/sites-available/default
+systemctl restart nginx
+```
+[nginx config](proxy/nginx.conf)
+
+# Update manually
+
+To update virtomize manually use the following command.
+
+**Always create a backup before updating.**
+
+```
+cd /opt/virtomize
+./update.sh -m
+```
+
+# Backup
+
+## Default Backups
+
+After starting using docker-compose some folders are created including the `backup` folder.
```

Comment:
After starting **Virtomize** using docker-compose**,** some folder are created**,** including the `backup` folder.  

Maybe specify the full backup path.